### PR TITLE
Added support for VersionPicker field in Jira.

### PR DESCRIPTION
**Pull request by MithileshPawar**
By submitting a PR to this repository, you agree to the terms within the [Checkmarx Code of Conduct](https://github.com/checkmarx-ts/open-source-template/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/checkmarx-ltd/open-source-template/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

> Adds support for new field of type VersionPicker in Jira.

### References

> #338 

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR (if applicable).  *If documentaiton is a Wiki Update, please indicate desired changes within PR MD Comment*
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used

### CODE REVIEW

**Comment by cxflowtestuser** [comment]
Scan submitted to Checkmarx

**Comment by cxflowtestuser** [comment]
### Checkmarx scan completed
[Full Scan Details](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1010510&projectid=176)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|112
Low|1403
Informational|51

#### Violation Summary
Severity|Count
---|---

#### Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---

File: src/main/java/com/checkmarx/flow/service/JiraService.java [hunk 1]
```diff
@@ -658,6 +659,20 @@ private void mapCustomFields(ScanRequest request, ScanResults.XIssue issue, Issu
...
+                            issueBuilder.setFieldValue(customField, ComplexIssueInputFieldValue.with(NAME_FIELD_TYPE, value));
+                            break;
+                        case "multi-version-picker":
+                            log.debug("multi version picker");
+                            String[] selectedVersions = StringUtils.split(value, ",");
+                            List<ComplexIssueInputFieldValue> versionList = new ArrayList<>();
+                            for (String version : selectedVersions) {
+                                ComplexIssueInputFieldValue fieldValue = ComplexIssueInputFieldValue.with(NAME_FIELD_TYPE, version.trim());
+                                versionList.add(fieldValue);
+                            }
```
Comment (alex-ko-dev) [inline src/main/java/com/checkmarx/flow/service/JiraService.java:673]: You can also use something like
```
List<ComplexIssueInputFieldValue> versionList = Arrays.stream(StringUtils.split(value, ","))
        .map(version -> ComplexIssueInputFieldValue.with(NAME_FIELD_TYPE, version.trim()))
        .collect(Collectors.toList());
```

**Comment by cxflowtestuser** [comment]
Scan submitted to Checkmarx

**Comment by cxflowtestuser** [comment]
### Checkmarx scan completed
[Full Scan Details](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1010540&projectid=176)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|112
Low|1403
Informational|51

#### Violation Summary
Severity|Count
---|---

#### Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---

**Comment by MithileshPawar** [comment]
@alex-ko-dev , Can you approve this again? I rebased the branch from develop.

File: src/main/java/com/checkmarx/flow/service/JiraService.java (hunk 1 above)
Comment (MithileshPawar) [reply src/main/java/com/checkmarx/flow/service/JiraService.java:673]: Alright !. For now, I just made use of the existing approach.

**Pull request merged**